packages/core-contracts/contracts/mocks/utils/ERC20Mock.sol

```diff
@@ -0,0 +1,20 @@
+//SPDX-License-Identifier: Unlicense
```

Comment:
ultra nit: but why have different versions of "Unlicense"/"UNLICENSED"?

Reply:
I am trying to fix it. The compiler suggests `UNLICENSED` in capital letters so I am replacing all occurrences with it.